core/templates/base-components/loading-message.component.ts

```diff
@@ -15,10 +15,18 @@
 /**
  * @fileoverview Component for showing the loading screen with a message.
  */
+import { Component, Input } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+  @Component({
+  selector: 'loading-message',
```

Comment:
Change the selector to `oppia-loading-message`. This would also require you to change the HTML tags in all the places from `loading-message` to `oppia-loading-message`